components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -1481,4 +1601,32 @@ private static List<String> getUserAttributeClaimMappingList(AuthenticatedUser a
                 })
                 .collect(Collectors.toList());
     }
+
+
+    private boolean isTrustedTokenIssuer(AuthenticationContext context) {
+
+        ExternalIdPConfig externalIdPConfig = context.getExternalIdP();
+        if (externalIdPConfig == null) {
+            return false;
+        }
+
+        IdentityProvider externalIdentityProvider = externalIdPConfig.getIdentityProvider();
+        if (externalIdentityProvider == null) {
+            return false;
+        }
+
+        IdentityProviderProperty[] identityProviderProperties = externalIdentityProvider.getIdpProperties();
+        for (IdentityProviderProperty identityProviderProperty: identityProviderProperties) {
+            if (identityProviderProperty.getName().equals(IdPManagementConstants.IS_TRUSTED_TOKEN_ISSUER)) {
```

Comment:
Use `<constant>.equals(<variable>)` pattern to avoid possible NPE

Reply:
Already addressed with https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/159/commits/95e4f9c1e60f1c6b6487c4d7a2b5e063922eaa15